.github/workflows/node.js.yml

```
@@ -0,0 +1,27 @@
+name: Build
+on:
+  push:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [17.x, 18.x]
+        os: [ubuntu-latest, windows-latest]
```

As discussed on Discord, we can probably do without the matrix given we’re just deploying to Netlify. Changing this here would still make it easy to add more Node versions if we want later:

```suggestion
        node-version: [16.x]
```